cfme/configure/access_control.py

```diff
@@ -304,6 +322,16 @@ def add_group_from_ldap_lookup(self):
              action=form_buttons.add)
         flash.assert_success_message('Group "{}" was saved'.format(self.description))
 
+    def add_group_from_ext_auth_lookup(self):
```

Comment:
force navigate is done at line 305, with "def retrieve_ext_auth_user_groups(self):"